joomla/joomla-cms
# Skip calling callback cache controller if caching is disabled

### Summary of Changes

No need to render a module via callback cache controller if caching is disabled.

### Testing Instructions

Enable Joomla caching, apply patch, see no changes except slight cached module rendering time improvement for logged users.

### Actual result BEFORE applying this Pull Request

The module is rendered or the module data is loaded via callback cache without storing the data, even if caching is disabled.

### Expected result AFTER applying this Pull Request

The callback is executed directly.

### Documentation Changes Required

No.
